perfkitbenchmarker/providers/ibmcloud/ibmcloud_virtual_machine.py

```diff
@@ -359,7 +359,7 @@ def DownloadPreprovisionedData(self, install_path, module_name, filename):
 
   def ShouldDownloadPreprovisionedData(self, module_name, filename):
     """Returns whether or not preprovisioned data is available."""
-    return True
+    return False
```

Comment:
Why was this set to False?

Reply:
It was set to True for a test before the code was submitted before, however that should have been set to False in the first place as we don't keep any pre provisioned data.

Reply:
Gotcha, thanks.